google/cloud/bigquery/table.py

```diff
@@ -2128,7 +2128,14 @@ def to_api_repr(self):
         return self._properties
 
     def _key(self):
-        return tuple(sorted(self._properties.items()))
+        properties = self._properties.copy()
```

Comment:
(nit)
It might be a good idea to explicitly explain in a short comment why shallow copy is sufficient here (i.e. we are only "renaming" top level keys).

Reply:
You are right, better be explicit here.